data_structures/src/transaction.rs

```diff
@@ -259,6 +280,36 @@ impl DRTransaction {
             .map(|tx_idx| TxInclusionProof::new(tx_idx, txs))
     }
 
+    /// Returns the weight of a data request transaction.
+    /// This is the weight that will be used to calculate
+    /// how many transactions can fit inside one block
+    pub fn weight(&self) -> u32 {
+        // DR_weight = DR_size*alpha + W*COMMIT + W*REVEAL*beta + TALLY*beta + W*OUTPUT_SIZE
+
+        let inputs_len = u32::try_from(self.body.inputs.len()).unwrap_or(u32::MAX);
+        let outputs_len = u32::try_from(self.body.outputs.len()).unwrap_or(u32::MAX);
+        let inputs_weight = inputs_len.saturating_mul(INPUT_SIZE);
+        let outputs_weight = outputs_len.saturating_mul(OUTPUT_SIZE);
```

Comment:
This will need to be changed after #1306, right?

Reply:
yes, later it would be only OUTPUT_SIZE